src/test/kotlin/extract/to/lib/jpt/report/Apdex.kt

```diff
@@ -0,0 +1,42 @@
+package extract.to.lib.jpt.report
+
+import com.atlassian.performance.tools.jiraactions.api.ActionMetric
+import extract.to.lib.jpt.report.Apdex.Satisfaction.FRUSTRATED
+import extract.to.lib.jpt.report.Apdex.Satisfaction.SATISFACTORY
+import extract.to.lib.jpt.report.Apdex.Satisfaction.TOLERATING
+import java.time.Duration
+
+class Apdex(
+    private val maxSatisfaction: Duration,
+    private val maxTolerance: Duration
+) {
+
+    constructor() : this(
+        Duration.ofSeconds(1),
+        Duration.ofSeconds(4)
+    )
```

Comment:
Why not default values in primary constructor?

Reply:
I have this Java API reflex. If we promote this to a lib, we'll have to avoid Kotlin default args.